frontend/src/stores/projectStore.js

```diff
@@ -64,6 +64,12 @@ const ProjectStore = Reflux.createStore({
       });
     }
     projects.id(id).fetch().then((project) => {
+      let sortedSprint = [];
+      _.each(project.currentSprint.tasks, (task) => {
+        sortedSprint[task.status] = sortedSprint[task.status] || [];
+        sortedSprint[task.status].push(task);
+      });
+      project.currentSprint.tasks = _.flatten(_.compact(sortedSprint));
```

Comment:
lines 67 - 72 looks the same as just this:

```
project.currentSprint.tasks = _.sortBy(project.currentSprint.tasks, (task) => {
  return task.status;
});
```

Note that lodash `_.sortBy` is stable